### PR TITLE
fix changelog hyperlink

### PR DESCRIPTION
**Pull request by jonahjon**
Signed-off-by: jonahjon <jonahjones094@gmail.com>

*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by panktishah26** [comment]
should we change this to main branch too?

File: docs/content/en/docs/reference/changelog.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ weight: 35
 - Add option to disable vSphere CSI driver [#3148](https://github.com/aws/eks-anywhere/issues/3148)
 - Add support for skipping load balancer deployment for Bare Metal so users can use their own load balancers [#3608](https://github.com/aws/eks-anywhere/pull/3608)
 - Add support to configure aws-iam-authenticator on workload clusters independent of management cluster [#2814](https://github.com/aws/eks-anywhere/issues/2814)
-- Add EKS Anywhere Packages support for remote management on workload clusters. (For more EKS Anywhere packages info: [v0.12.0](https://github.com/aws/eks-anywhere-packages/releases/tag/v0.12.0))
+- Add EKS Anywhere Packages support for remote management on workload clusters. (For more EKS Anywhere packages info: [v0.12.0](https://github.com/aws/eks-anywhere-packages/releases/tag/v0.2.14))
```
Comment (TerryHowe) [inline docs/content/en/docs/reference/changelog.md:22]: Click v0.12.0 and get v0.2.14! I can see why you did that, but confusing

**Pull request merged**